calcite-adapter/src/main/java/software/amazon/documentdb/jdbc/calcite/adapter/DocumentDbRules.java

```diff
@@ -333,24 +333,30 @@ private static Operand getMongoAggregateForAndOperator(final RexCall call, final
             sb.deleteCharAt(sb.length() - 1);
             sb.append("]}, false, null]}]}");
 
-            return new Operand(sb.toString());
+            return new Operand(
```

Comment:
Can support AND in simple match BUT this can only be nested in other ANDs and ORs not in a comparison operator so isQuerySyntax is false here